blockly-core/core/ui/contract_editor/contract_editor.js

```diff
@@ -310,10 +310,22 @@ Blockly.ContractEditor.prototype.moveExampleBlocksToModal_ = function (functionN
   var exampleBlocks = Blockly.mainBlockSpace.findFunctionExamples(functionName);
   exampleBlocks.forEach(function(exampleBlock) {
     var movedExampleBlock = this.moveToModalBlockSpace_(exampleBlock);
+    var exampleCall = movedExampleBlock.getInputTargetBlock(Blockly.ContractEditor.EXAMPLE_BLOCK_ACTUAL_INPUT_NAME);
+    exampleCall.setMovable(false);
+    exampleCall.setDeletable(false);
     this.exampleBlocks.push(movedExampleBlock);
   }, this);
 };
 
+/**
+ * @override
+ */
+Blockly.ContractEditor.prototype.moveToModalBlockSpace_ = function (block) {
+  var newBlock = Blockly.ContractEditor.superClass_.moveToModalBlockSpace_.call(this, block);
```

Comment:
something_.somethingelse_ makes it look like we're accessing things we shouldn't be. I would think there's a proper closure way to access super class?